reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -2717,4 +2717,40 @@ void testMaxKeepAliveRequestsBadValues(int maxKeepAliveRequests) {
 				.isThrownBy(() -> createServer().maxKeepAliveRequests(maxKeepAliveRequests))
 				.withMessage("maxKeepAliveRequests must be positive or -1");
 	}
+
+	@Test
+	void testIsFormUrlencodedWithCharset() {
+		doTestIsFormUrlencoded("application/x-www-form-urlencoded;charset=UTF-8", true);
+	}
+
+	@Test
+	void testIsFormUrlencodedWithoutCharset() {
+		doTestIsFormUrlencoded("application/x-www-form-urlencoded", true);
+	}
+
+	@Test
+	void testIsNotFormUrlencoded() {
```

Comment:
maybe add a test that `application/x-www-form-urlencoded-bad` is not `isFormUrlencoded`?